addons/ica_web_responsive/static/src/webclient/share_url/share_url.js

```
@@ -0,0 +1,42 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { markup } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { browser } from "@web/core/browser/browser";
+import { isDisplayStandalone } from "@web/core/browser/feature_detection";
+import { escape } from "@web/core/utils/strings";
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)